lib/tasks/install_plugin.rake

```diff
@@ -0,0 +1,64 @@
+require 'fileutils'
+
+PLUGIN_NAME = 'redmine_tracky'
+LINE_SEPARATOR_LENGTH = 60
+
+namespace :redmine do
+  namespace :plugins do
+    namespace :"#{PLUGIN_NAME}" do
+      @plugin = PLUGIN_NAME
+
+      task :install do
+        Rake::Task["redmine:plugins:migrate"].invoke
+        Rake::Task["redmine:plugins:#{PLUGIN_NAME}:convert:sass"].invoke
```

Comment:
The guard file is here to help you :D If you change any css/scss file in plugins/redmine_tracky/assets/stylesheets it will rexport all scss files to the assets folder

Reply:
Like webpacker hot-reload :D

Reply:
`so it's compiled on install. what if we deploy new versions or update it?`\

I'll look into that 🤔